### ISSUE TITLE
Create a QuickVega.jl package

### ISSUE THREAD

**Issue by davidanthoff** [queryverse/VegaLite.jl#265]
The idea would be to have an alternative API in the [QuickVega.jl](https://github.com/queryverse/QuickVega.jl) package that is not grammar-of-graphics based, but resembles the original API from https://github.com/johnmyleswhite/Vega.jl, i.e. the philosophy would be "one super simple function call to generate one type of plot".

In my mind we would have things like `scatter(x, y)` etc.

I think this package could also have functions like the ones in seaborn, i.e. really powerful functions that create quite complicated figures with one call.

The output of each of these function calls would be a `VLSpec` (or `VGSpec`) from this package here, i.e. we would reuse the whole display/save etc. functionality. 

Once we have https://github.com/queryverse/VegaLite.jl/issues/242, I think this could become a _really_ powerful and generic combination. For example, one could do things like
```julia
QuickVega.powerfull_fancy_something(...) |> @vlmod(title="NEW TITLE")
```
i.e. one could start with a template generated with a function in QuickVega.jl, and then get all the powerful manipulation options from a `@vlmod` macro (or even a `@vgmod` macro).

**Comment by ym-han** [comment]
Was there supposed to be a link to a package in "The output of each of these function calls would be a VLSpec (or VGSpec) from this package here, i.e. we would reuse the whole display/save etc. functionality"?

**Comment by davidanthoff** [comment]
No, so the idea would be that QuickVega.jl has a dependency on Vega.jl and VegaLite.jl, and the functions that are defined in QuickVega.jl return instances of either `VLSpec` or `VGSpec`.

**Comment by ym-han** [comment]
Right, gotcha!

**Comment by pablosanjose** [comment]
I think this and the associated `@vlmod` idea would be incredibly useful. The barrier of entry of VegaLite is not particularly low, I must say. Incremental construction/modification of plots might be easier to grasp and extend for many. Also, if I write VegaLite recipe for a type of mine, a user might easily want to tweak it, which the `@vlmod` approach would allow her to do. Otherwise, she'd be forced to dive into the recipe's code and tweak it directly.

**Comment by ym-han** [comment]
I plan to start working on QuickVega either this weekend or sometime next week. It doesn't sound like it'd be hard to implement at least the basic functionality.
